src/components/MultiChoice/index.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { act, render, screen, within } from '@testing-library/react';
+
+import { answerFabricator } from 'tests/fabricator';
+import { Answer } from 'types/answer';
+
+import MultiChoice, { multiChoiceDataTestIds } from '.';
+
+describe('MultiChoice', () => {
+  const answers = answerFabricator.times(3);
+
+  it('renders a multi choice component', () => {
+    const onValuesChanged = () => {
+      // Do nothing
+    };
+    render(<MultiChoice items={answers} isPickOne={false} onValuesChanged={onValuesChanged} />);
+
+    const multiChoice = screen.getByTestId(multiChoiceDataTestIds.base);
+
+    expect(multiChoice).toBeVisible();
+    expect(within(multiChoice).getAllByRole('presentation')).toHaveLength(3);
+  });
+
+  describe('given can pick many choices', () => {
+    describe('given the first two items is clicked', () => {
+      it('renders the first two components as checked state', () => {
+        let selectedValues: Answer[] = [];
+
+        const handleValuesChanged = (answers: Answer[]) => {
+          selectedValues = answers;
+        };
+
+        render(<MultiChoice items={answers} isPickOne={false} onValuesChanged={(answers) => handleValuesChanged(answers)} />);
+
+        const multiChoice = screen.getByTestId(multiChoiceDataTestIds.base);
+
+        act(() => {
+          within(multiChoice).getAllByRole('presentation').at(0)?.click();
+        });
+
+        act(() => {
+          within(multiChoice).getAllByRole('presentation').at(1)?.click();
+        });
+
+        expect(within(multiChoice).getByText(answers[0].text)).not.toHaveClass('opacity-50');
```

Comment:
Could you add a comment to explain what is 'opacity-50' class?

Reply:
@hoangmirs added a comment in [9cf9cc7](https://github.com/manh-t/react-survey/pull/79/commits/9cf9cc7e64b19e65e275327d6b0981dc4379edc3)